src/citra_qt/configuration/configure_general.cpp

```diff
@@ -56,6 +59,21 @@ ConfigureGeneral::ConfigureGeneral(QWidget* parent)
                     .rightJustified(tr("unthrottled").size()));
         }
     });
+
+    connect(ui->screenshot_dir_path, &QLineEdit::editingFinished, [&] {
+        const QString path = ui->screenshot_dir_path->text();
+        UISettings::values.screenshot_path = path;
+    });
+
+    connect(ui->change_screenshot_dir, &QPushButton::clicked, this, [this] {
+        const QString dir_path = QFileDialog::getExistingDirectory(
+            this, tr("Select Screenshot Directory"), UISettings::values.screenshot_path,
+            QFileDialog::ShowDirsOnly);
+        if (!dir_path.isEmpty()) {
+            UISettings::values.screenshot_path = dir_path;
+            SetConfiguration();
```

Comment:
See above. This should just be
```suggestion
            ui->screenshot_dir_path->setText(dir_path);
```